Analysis/Tasks/PID/pidTPC_tiny.cxx

```diff
@@ -242,7 +266,7 @@ WorkflowSpec defineDataProcessing(ConfigContext const& cfgc)
 {
   auto workflow = WorkflowSpec{adaptAnalysisTask<pidTPCTaskTiny>(cfgc, TaskName{"pidTPC-tiny-task"})};
   if (cfgc.options().get<int>("add-qa")) {
-    workflow.push_back(adaptAnalysisTask<pidTPCTaskQA>(cfgc, TaskName{"pidTOFQA-task"}));
+    workflow.push_back(adaptAnalysisTask<pidTPCTaskQA>(cfgc, TaskName{"pidTPCQA-task"}));
```

Comment:
can we drop TaskName?

Reply:
Well yes we do but having `TOF` in the task name means that it gets parsed into `t-o-f` that's the only reason that I would like to keep it

Reply:
OK. Then let's keep it